core/request.go

```diff
@@ -104,11 +106,12 @@ func (r *RequestAccessor) StripBasePath(basePath string) string {
 
 // ProxyEventToHTTPRequest converts an API Gateway proxy event into an
 // http.Request object.
-// Returns the populated request with an additional two custom headers for the
-// stage variables and API Gateway context. To access these properties use
-// the GetAPIGatewayStageVars and GetAPIGatewayContext method of the RequestAccessor
-// object.
-func (r *RequestAccessor) ProxyEventToHTTPRequest(req events.APIGatewayProxyRequest) (*http.Request, error) {
```

Comment:
I think you should add a separate method for this, as you did with the other adapters to avoid changing the exported API. It could get a shorter name, like `ConvertEventToRequest` or `RequestFromEvent` or some such.

Reply:
(the rationale being that if it's exported, someone can be depending on the old signature).

Reply:
Yes, good point. Will refactor.